src/constants.ts

```diff
@@ -9,6 +9,8 @@ export const SESSION_KEYS = {
   EMAIL: 'auth:email',
   TOTP: 'auth:totp',
   TOTP_EXPIRES_AT: 'auth:totp:expiresAt',
+  FIRST_OTP_REQUESTED: 'auth:totp:firstOtpRequested',
```

Comment:
For consistency, we are calling these constants `TOTP`; a rename would be required.